logging.go

```diff
@@ -0,0 +1,24 @@
+package gojenkins
+
+import (
+	"log"
+)
+
+
+func Info (message string) {
+
+	log.Printf("INFO: %s",message) 
+
+}
+
+func Warning (message string) {
+
+	log.Printf("WARNING: %s",message)
+
+}
+
+func Error (message string) {
+
+	log.Printf("ERROR: %s",message)
```

Comment:
Compared with the old implementation, this can not output info/warning log and error log to different standard files, maybe it is not good enough.

Reply:
Opps... Sorry I was moving forward did not mean for it to be part of this PR